packages/p2p/src/components/buy-sell/buy-sell-row.scss

```diff
@@ -90,6 +90,13 @@
 
     &__button {
         align-self: flex-start;
+        display: flex;
+        flex-flow: row wrap;
+        white-space: inherit;
+        height: fit-content;
+        max-height: 7rem;
+        margin-left: auto;
```

Comment:
Set `margin-left: auto` to prevent button from centering automatically due to flex, and to adjust button width to be fitted according to the content:

### Before
<img width="488" alt="Screenshot 2022-07-18 at 3 30 52 PM" src="https://user-images.githubusercontent.com/103016120/179463609-ed336a8a-7c6c-4781-8938-5bf69ca99579.png">

### After:
<img width="484" alt="Screenshot 2022-07-18 at 3 30 56 PM" src="https://user-images.githubusercontent.com/103016120/179463629-b9f3bc90-4423-4a6a-82d1-c2fca0ee26bb.png">

Reply:
Refactored code to use flex noww